frontend/packages/console-shared/src/components/formik-fields/index.ts

```diff
@@ -1,12 +1,16 @@
-export { default as InputField } from './InputField';
-export { default as TextAreaField } from './TextAreaField';
-export { default as DropdownField } from './DropdownField';
-export { default as NSDropdownField } from './NSDropdownField';
 export { default as CheckboxField } from './CheckboxField';
-export { default as NumberSpinnerField } from './NumberSpinnerField';
-export { default as EnvironmentField } from './EnvironmentField';
+export { default as DropdownField } from './DropdownField';
 export { default as DroppableFileInputField } from './DroppableFileInputField';
+export { default as EnvironmentField } from './EnvironmentField';
+export { default as InputField } from './InputField';
 export { default as InputSearchField } from './InputSearchField';
 export { default as MultiColumnField } from './multi-column-field/MultiColumnField';
-export { default as ResourceLimitField } from './ResourceLimitField';
+export { default as NSDropdownField } from './NSDropdownField';
+export { default as NumberSpinnerField } from './NumberSpinnerField';
 export { default as RadioButtonField } from './RadioButtonField';
+export { default as ResourceDropdownField } from './ResourceLimitField';
+export { default as ResourceLimitField } from './ResourceLimitField';
+export { default as SwitchField } from './SwitchField';
+export { default as TextAreaField } from './TextAreaField';
+export * from './field-utils';
+export * from './field-types';
```

Comment:
I am slightly concerned if we really want to re-export these. It allows to use '@console/shared' for imports but we could hit naming collisions/confusion as the `@console/shared` package grows. Should we re-export everything which is used outside the shared package? Should we re-export only selected components and utils? Should we split shared package into multiple packages (e.g. @console/shared-forms) to avoid potential naming conflicts?

Reply:
We could definitely create more packages for isolated use, but we need to be very clear with who is dependant on who otherwise we run the risk of circular references. I wonder if we could just export at a deeper level... something like `@console/shared/src/formik` or something... 

@christianvogt @spadgett thoughts?

Reply:
Perhaps we circle back to this - you may have a growing issue with rebases as time goes on. Recommend we get this in and circle back to address the export.